src/data/content/homepage.ts

```diff
@@ -5,82 +5,50 @@ export default {
-        },
-      ],
+  platformProducts: {
+    title: 'Platform and Products',
+    platformCard: {
+      title: 'Platform',
+      content: "Understand the core concepts and features of Ably's platform.",
+      image: 'platform.png',
+      link: '/docs/platform',
     },
-    {
-      columns: 2,
-      bottomMargin: 80,
-      cards: [
-        {
-          title: 'Platform',
-          type: 'feature',
-          content: "Understand the core concepts of Ably's platform, its pricing and integrations.",
-          image: 'platform.png',
-          links: [{ text: 'Get started', href: '/docs/platform' }],
-        },
-        {
-          title: 'Pub/Sub',
-          type: 'feature',
-          content: "Quickly create realtime digital experiences with Ably's core building blocks.",
-          image: 'pub-sub.png',
-          links: [{ text: 'Get started', href: '/docs/basics' }],
-        },
-        {
-          title: 'Chat',
-          type: 'feature',
-          content: 'Quickly add chat features into any application.',
-          image: 'chat.png',
-          links: [{ text: 'Get started', href: '/docs/chat' }],
-        },
-        {
-          title: 'Spaces',
-          type: 'feature',
-          content: 'Build collaborative environments in just a few lines of code.',
-          image: 'spaces.png',
-          links: [{ text: 'Get started', href: '/docs/spaces' }],
-        },
-        {
-          title: 'LiveSync',
-          type: 'feature',
-          content: 'Seamlessly sync database changes with frontend clients.',
-          image: 'livesync.png',
-          links: [{ text: 'Get started', href: '/docs/livesync' }],
-        },
-        {
-          title: 'Asset Tracking',
-          type: 'feature',
-          content: 'Track the location of assets in realtime.',
-          image: 'asset-tracking.png',
-          links: [{ text: 'Get started', href: '/docs/asset-tracking' }],
-        },
-      ],
-    },
-    {
-      columns: 1,
-      bottomMargin: 160,
-      cards: [
-        {
-          title: 'SDKs',
-          content: 'Ably SDKs provide a consistent and idiomatic API across a variety of supported platforms.',
-          image: 'sdks@2x.png',
-          type: 'sdk',
-          callToAction: { text: 'View all SDKs', href: '/docs/sdks' },
-        },
-      ],
-    },
-  ],
+    productCards: [
+      {
+        name: 'pubsub',
+        link: '/docs/basics',
+      },
+      {
+        name: 'chat',
+        link: '/docs/chat',
+      },
+      {
+        name: 'spaces',
+        link: '/docs/spaces',
+      },
+      {
+        name: 'liveSync',
+        link: '/docs/livesync',
+      },
+      {
+        name: 'assetTracking',
+        link: '/docs/asset-tracking',
+      },
+      {
+        name: 'liveObjects',
+        link: '/docs/objects',
+      },
+    ],
+  },
+  examples: {
+    title: 'Examples',
+    link: { text: 'View all', href: '/docs/examples' },
```

Comment:
```suggestion
    link: { text: 'View all', href: '/examples' },
```

I think this was lost in the rebase too.

Reply:
Yes, that's what I suggested. It reverted to `/docs/examples` in the rebase